lms/static/js/fixtures/search_form.html

```diff
@@ -0,0 +1,7 @@
+<div id="courseware-search-bar" class="search-container">
+    <form role="search-form">
```

Comment:
@mattdrayer, @chrisndodge, @cdodge, There is no such role as "search-form".  Must be removed.